src/components/SearchAndFilter/SearchAndFilter.js

```diff
@@ -9,7 +9,7 @@ import { overflowingChipsCount, isChipInArray } from "./shared";
 import "./SearchAndFilter.scss";
 
 const SearchAndFilter = ({
-  filterPanelData,
+  filterPanelData = [],
```

Comment:
The prop type for this has `isRequired` so there are a bunch of prop warnings for this and `returnSearchData` on the storybook page.

Reply:
Good catch, thanks. Now fixed.